flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/AwsV2Util.java

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
You've probably mitigated this (and I just can't figure it out yet): but is it a worry that anyone using `AwsV2Util` as a dependency will possibly see some of their code break as a result of them not being able to access certain `AwsV2Util` methods that were previously in this file? Such as maybe `AwsV2Util. createKinesisAsyncClient()`?